conduit-spotify/src/Commands/Focus.php

```
@@ -107,20 +118,47 @@ public function handle(AuthInterface $auth, ApiInterface $api): int
         $this->newLine();
 
+        // Show recommendation first if available
+        $recommended = $this->getRecommendedFocusMode();
+        if ($recommended && isset($presets[$recommended])) {
+            $emoji = $this->getFocusEmoji($recommended);
+            $description = $this->getFocusDescription($recommended);
+            $this->line("  🌟 <info>{$recommended}</info> - {$description} <fg=green>(Recommended)</fg=green>");
+            $this->newLine();
+        }
+
         foreach ($presets as $mode => $uri) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$uri'. (Unused Code Rules)

(UnusedLocalVariable)